policyengine_us/reforms/reconciliation/reconciled_qbid.py

```diff
@@ -17,35 +17,82 @@ class qbid_amount(Variable):
 
         def formula(person, period, parameters):
             p = parameters(period).gov.irs.deductions
-
-            # 1. Compute the new maximum QBID
+            p_ref = parameters(period).gov.contrib.reconciliation.qbid
+            
+            # Basic inputs
             qbi = person("qualified_business_income", period)
-            qbid_max = p.qbi.max.rate * qbi
-
-            # 2. Compute the wage/property cap (unchanged)
+            is_sstb = person("business_is_sstb", period)  # Boolean indicator
+            
+            # Split QBI based on SSTB status (simplified: all or none)
+            qbi_sstb = where(is_sstb, qbi, 0)
```

Comment:
Our convention, though are these variables used anywhere?
```suggestion
            qbi_sstb = is_sstb * qbi
```